front-end/src/app/transactions/transaction-list/transaction-list.component.html

```diff
@@ -111,7 +111,32 @@ <h5 class="m-0">Transactions during coverage dates</h5>
           {{ item.parent_transaction?.transaction_id }}
         </td>
         <td>
-          <p-button pRipple icon="pi pi-ellipsis-v" styleClass="p-button-text" ariaLabel="action"></p-button>
+          <p-overlayPanel #op [styleClass]="'action-options'">
```

Comment:
I wonder if there's a clean way to add this into the generic table component

Reply:
Yeah i think that would get pretty big.  Maybe a compromise would be to turn the ellipsis and dropdown into a component.  The callbacks would still work because we bound them to the list component. No blocker though

Reply:
Yes, let's pull the p-overlay with button HTML out into a component. At the very least it will reduce the HTML code duplication in the transaction-list component and I'm sure will be used elsewhere.

`<p-overlayPanel #op [styleClass]="'action-options'">
            <ng-template pTemplate>
              <ul>
                <li *ngFor="let action of rowActions">
                  <button
                    *ngIf="!action.isAvailable || action.isAvailable(item)"
                    pButton
                    pRipple
                    type="button"
                    class="p-button-secondary p-button-text"
                    id="one"
                    (click)="onRowActionClick(action, item)"
                  >
                    {{ action.label }}
                  </button>
                </li>
              </ul>
            </ng-template>
          </p-overlayPanel>
          <p-button 
            pRipple 
            icon="pi pi-ellipsis-v" 
            styleClass="p-button-text"
            (onClick)="op.toggle($event)"
            ariaLabel="action"
          ></p-button>
          <!-- <em class="pi pi-trash" (click)="deleteItem(item)"></em> -->
        </td>`